src/components/context-provider/context-provider.tsx

```diff
@@ -0,0 +1,33 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import {omit} from '../../utils';
+
+export interface CPProps {
```

Comment:
maybe name it `ContextProviderProps` — i don't see any reason to make acronym here.

Reply:
`+`